mlir-tensorrt/compiler/lib/Dialect/Plan/Transforms/OutlineClusters.cpp

```diff
@@ -264,6 +266,13 @@ static LogicalResult outlineTensorRTRegion(RewriterBase &rewriter,
   return success();
 }
 
+static LogicalResult outlineTensorRTRegion(RewriterBase &rewriter,
+                                           plan::InlineClosedGroupNonDPSOp op) {
+  llvm_unreachable("outlinining non-dps inline closed group ops to tensorrt "
```

Comment:
when `llvm_unreachable` is used, it results in a crash/abort, which is not desired. Error should be propagated to callers so that the pass can signal failure. Just `return op->emitError(.....)` should be sufficient here.